### PR TITLE
doc: update news

### PR DESCRIPTION
**Pull request by komainu8**

### CODE REVIEW

File: doc/source/news.rst [hunk 1]
```diff
@@ -7,6 +7,49 @@
...
+
+Release 8.0.0 - 2018-02-09
+--------------------------
+
+Improvements
+^^^^^^^^^^^^
+
+* [:doc:`/reference/commands/select`] Added ``--drilldown_adjuster`` and
+  ``--drilldowns[LABEL].adjuster``.
+  You can score adjust against result of drilldown.
```
Comment (kou) [inline doc/source/news.rst]: "score adjust" -> "adjust score"

File: doc/source/news.rst [hunk 2]
```diff
@@ -7,6 +7,49 @@
...
+
+* [:ref:`online-index-construction`] Changed environment variable name
+  ``GRN_II_REDUCE_EXPIRE_ENABLE`` to ``GRN_II_REDUCE_EXPIRE_THRESHOLD``.
+
+  GRN_II_REDUCE_EXPIRE_THRESHOLD=0 == GRN_II_REDUCE_EXPIRE_ENABLE=no.
```
Comment (kou) [inline doc/source/news.rst]: Markup is missing.

```rest
``...``
```

File: doc/source/news.rst [hunk 3]
```diff
@@ -7,6 +7,49 @@
...
+  GRN_II_REDUCE_EXPIRE_THRESHOLD=-1 (default) uses
```
Comment (kou) [inline doc/source/news.rst]: The default is 32 now.

File: doc/source/news.rst [hunk 4]
```diff
@@ -7,6 +7,49 @@
...
+  ii->chunk->max_map_seg / 2 as threshold.
+  GRN_II_REDUCE_EXPIRE_THRESHOLD > 0 uses
+  MIN(ii->chunk->max_map_seg / 2, GRN_II_REDUCE_EXPIRE_THRESHOLD)
+  as threshold.
+
+* [:doc:`/reference/functions/between`] Accept between() without borders.
+  If the number of arguments passed to between() is 3, the 2nd and 3rd
+  arguments are handled as the inclusive edges.[GitHub#685]
```
Comment (kou) [inline doc/source/news.rst]: A space is missing after `.`.

File: doc/source/news.rst [hunk 5]
```diff
@@ -7,6 +7,49 @@
...
+
+Fixes
+^^^^^
+
+* Fixed a memory leak for not tiny hash table.
```
Comment (kou) [inline doc/source/news.rst]: "not tiny" -> "normal".

File: doc/source/news.rst [hunk 6]
```diff
@@ -7,6 +7,49 @@
...
+  [GitHub:mroonga/mroonga#190][Reported by fuku1]
+
+* Fix a memory leak for not tiny array.
```
Comment (kou) [inline doc/source/news.rst]: ditto.

File: doc/source/news.rst [hunk 7]
```diff
@@ -7,6 +7,49 @@
...
+
+* [:doc:`/reference/commands/select`] Don't cache when output_columns uses not stable function.
```
Comment (kou) [inline doc/source/news.rst]: "Don't" -> "Stopped to"

Comment (kou) [reply doc/source/news.rst]:
```rest
``output_columns``
```

**Comment by komainu8** [comment]
@kou Thank you for your comments.
I've fixed them.

**Comment by kou** [comment]
Merged with a `GRN_II_REDUCE_EXPIRE_THRESHOLD` fix.

**Pull request merged**